src/shared/pages/team/teamView/teamView.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { Reducer, useEffect } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { History } from 'history';
+import { deleteUserFromTeam, getTeamMembers } from '../../../services/usersService';
+import { getTeam } from '../../../services/teamsService';
+import { State } from './state';
+import { Action } from './actions';
+import { reducer } from './reducer';
+import { initialState } from './initialState';
+import { User } from '../../../models/user';
+import { GENERAL_ERROR_TITLE, LOAD_TEAM_ERROR_DESCRIPTION, LOAD_TEAM_MEMBERS_ERROR_DESCRIPTION } from '../../../models/constants';
+import ErrorSummary from '../../../common/components/errorSummary';
+
+interface MatchParams {
+    teamId: string;
+}
+
+interface TeamMembersProps extends RouteComponentProps<MatchParams> {
+    history: History;
+}
+
+const onAddTeamMembersAddClick = (history: History, teamId: string) => {
+    history.push(`/team/${teamId}/add-users`);
+};
+
+const onBackLinkClick = (history: History) => {
+    history.push('/team_search');
+};
+
+const TeamView: React.FC<TeamMembersProps> = ({ history, match }) => {
+
+    const [state, dispatch] = React.useReducer<Reducer<State, Action>>(reducer, initialState);
+
+    const { params: { teamId } } = match;
+
+    useEffect(() => {
+        getTeam(teamId)
+            .then(team => dispatch({ type: 'SetTeamName', payload: team.displayName }))
+            .catch(() => {
+                dispatch({ type: 'SetGeneralError', payload: { description: LOAD_TEAM_ERROR_DESCRIPTION, title: GENERAL_ERROR_TITLE } });
+            });
+        getTeamMembers(teamId)
+            .then((users: User[]) => dispatch({ type: 'PopulateTeamMembers', payload: users }))
+            .catch(() => {
+                dispatch({ type: 'SetGeneralError', payload: { description: LOAD_TEAM_MEMBERS_ERROR_DESCRIPTION, title: GENERAL_ERROR_TITLE } });
+            });
+    }, []);
+
+    const removeTeamMember = (userUUID: string, teamId: string) => {
+        deleteUserFromTeam(userUUID, teamId)
+            .then(() => {
+                getTeamMembers(teamId)
+                    .then((users: User[]) => dispatch({ type: 'PopulateTeamMembers', payload: users }))
+                    .catch(() => {
+                        dispatch({ type: 'SetGeneralError', payload: { description: LOAD_TEAM_MEMBERS_ERROR_DESCRIPTION, title: GENERAL_ERROR_TITLE } });
+                    });
+            });
+    };
+
+    const DisplayTeamTable = () => (
+        <div>
```

Comment:
are these two divs needed? or do they need to have the row/column classes?

Reply:
Divs removed ad wrapped in <Fragment>